inference-engine/tests/functional/plugin/multi/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME multiFuncTests)
```

Comment:
if it's OK for you, we can put MULTI tests in `inference-engine/tests/functional/inference_engine/multi` since MULTI as well as HETERO are common IE functionality.